### ISSUE TITLE
404

### ISSUE THREAD

**Issue by josephgupkim** [clevyclev/Jammming#2]
I set up your project on my machine and tried to get the functionalities working but I'm running into a 404: Bad Request.

This meant that the client was able to communicate with the server, but the server could not find the requested file.

**Comment by clevyclev** [comment]
Hi Joseph.  Do you mean when you try to search for something?  That is an error that I am still trying to debug, and so far it is getting the best of me.

**Comment by josephgupkim** [comment]
Yes correct. Alright. Feel free to just re-notify me when you get this functionality running! I may recommend looking at a fellow peer's code and compare where you can possible improve yours.

**Comment by clevyclev** [comment]
Will do.  I think that I have isolated my issue to something in my JSON
response (it comes back as "ok: false), but I am not sure how to break it
down in order to debug it.
